vscode-extension/src/extension.ts

```diff
@@ -1,12 +1,13 @@
 // tslint:disable:no-var-requires
 // tslint:disable:no-require-imports
 // tslint:disable:no-use-before-declare
+// tslint:disable:no-shadowed-variable
```

Comment:
Do we really need to disable this?

Reply:
Yes for now I need to disable this, since travis would fail. But with the refactor the functions will be renamed anyway, so I'll remove it again in the refactor branch

Reply:
Alright.